br/pkg/task/backup_test.go

```diff
@@ -222,3 +223,27 @@ func TestBackupConfigHash(t *testing.T) {
 		hashCheck(t, &testCfg, originalHash, true)
 	}
 }
+
+func TestDefaultBackupConfigDisableChecksum(t *testing.T) {
+	// Test the default configuration
+	cfg := DefaultBackupConfig()
+
+	// Check some default values
+	require.Equal(t, uint32(4), cfg.Concurrency)
+	require.Equal(t, uint32(2), cfg.ChecksumConcurrency)
+	require.False(t, cfg.SendCreds)
+	require.False(t, cfg.Checksum)
+
+	// Test with checksum flag set
+	os.Args = []string{"cmd", "--checksum=true"}
+	cfg = DefaultBackupConfig()
+	require.True(t, cfg.Checksum)
```

Comment:
I don't think the `DefaultBackupConfig` will parse command line from `os.Args`... TBH I'm even not sure how this case passes...

Reply:
yeah you are absolutely right, should revisit before making PR read for review. It did pass locally and on CI which is weird.

Reply:
That's because tests in `br/pkg/task` is skipped on CI.
https://github.com/pingcap/tidb/blob/c550aae6a125e20c2aaa8d4f11a2f090bc8b2b50/Makefile#L627